x/merkledb/intermediate_node_db_test.go

```diff
@@ -125,3 +125,70 @@ func TestIntermediateNodeDB(t *testing.T) {
 	require.NoError(it.Error())
 	require.Equal(added+1, count)
 }
+
+func FuzzIntermediateNodeDBConstructDBKey(f *testing.F) {
+	f.Fuzz(func(
+		t *testing.T,
+		key []byte) {
```

Comment:
The spacing just looks pretty gross to me... not sure if this is normal for merkledb tho 🤷 
```suggestion
	f.Fuzz(func(t *testing.T, key []byte) {
```